emain/emain-window.ts

```
@@ -73,11 +73,32 @@ export class WaveBrowserWindow extends BaseWindow {
+
+        if (
+            (winWidth == null || winWidth === 0 || winHeight == null || winHeight === 0) &&
+            settings?.["window:dimensions"]
+        ) {
+            const dimensions = settings["window:dimensions"];
+            const [dimensionWidth, dimensionHeight] = dimensions.split(/[xX]/);
+
+            const parsedWidth = parseInt(dimensionWidth);
+            const parsedHeight = parseInt(dimensionHeight);
+
+            if ((!winWidth || winWidth === 0) && !isNaN(parsedWidth)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)